src/aiperf/endpoints/openai_chat.py

```
@@ -40,66 +74,139 @@ async def format_payload(
+        Returns:
+            Extracted response data or None if no content
+        """
+        match json_obj.get("object"):
+            case "chat.completion":
+                data_key = "message"
+            case "chat.completion.chunk":
+                data_key = "delta"
+            case _:
+                raise ValueError(
+                    f"Unsupported OpenAI object type: {json_obj.get('object')}: {json_obj}"
+                )
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)